datadog/resource_datadog_synthetics_global_variable.go

```diff
@@ -0,0 +1,143 @@
+package datadog
+
+import (
+	"strings"
+
+	datadogV1 "github.com/DataDog/datadog-api-client-go/api/v1/datadog"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceDatadogSyntheticsGlobalVariable() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDatadogSyntheticsGlobalVariableCreate,
+		Read:   resourceDatadogSyntheticsGlobalVariableRead,
+		Update: resourceDatadogSyntheticsGlobalVariableUpdate,
+		Delete: resourceDatadogSyntheticsGlobalVariableDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"tags": {
+				Type:     schema.TypeList,
+				Required: true,
```

Comment:
Is it really required? I would assume an empty list works.

Reply:
Yes it works with an empty list, but it needs to be set (like for synthetics tests). I suppose we could default to an empty list if nothing is specified in the config.